pkg/ipamd/ipamd.go

```diff
@@ -374,9 +376,15 @@ func New(rawK8SClient client.Client, cachedK8SClient client.Client) (*IPAMContex
 	c.enableIPv4 = isIPv4Enabled()
 	c.enableIPv6 = isIPv6Enabled()
 
+	eventRecorder, err := eventrecorder.New(rawK8SClient, cachedK8SClient)
+ 	if err != nil {
+ 		return nil, errors.Wrap(err, "ipamd: can not initialize event recorder")
```

Comment:
nit: `can not` --> `unable to`. `can not` sounds like a permission related error to me.